### PR TITLE
CB2-1584 Manually trigger lambda function with optional date.

### PR DESCRIPTION
**Pull request by Ian-Hodges**

### CODE REVIEW

File: src/handler.ts [hunk 1]
```diff
@@ -11,11 +12,18 @@ console.log(
   `\nRunning Service:\n '${SERVICE}'\n mode: ${NODE_ENV}\n stage: '${AWS_PROVIDER_STAGE}'\n region: '${AWS_PROVIDER_REGION}'\n\n`,
 );
 
-const handler = async (event: APIGatewayEvent): Promise<{ statusCode: number; body: string }> => {
-  console.log(`Function triggered by event: ${event.httpMethod}`);
-
+const handler = async (event: any): Promise<{ statusCode: number; body: string }> => {
```
Comment (premghinde) [inline src/handler.ts:15]: Shouldn't be regressing to `any`. Create a custom type if you need to.

Comment (danielchildsbjss) [reply src/handler.ts:15]: I think you can mark it as optional 
`APIGatewayEvent?`

**Review by danielchildsbjss — APPROVED** [review]

File: src/handler.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```
Comment (premghinde) [inline src/handler.ts:1]: remove?

**Review by premghinde — APPROVED** [review]

File: src/handler.ts (hunk 2 above)
Comment (Ian-Hodges) [reply src/handler.ts]: Yes, just spotted that too.

**Pull request merged**